spec/visual/samples/cfscript

```diff
@@ -124,6 +124,25 @@ xyz = {
     }
   ]
 }
+//syntax issue #459 (inherited from JS lexer, related to objects/properties)
+var doc = {
+  "first": {
+    "key": "value"
+  },
+  "hello": "world",
+};
+
+var options = {
+  vAxes: {
+    0: { title: "Rainfall (mm)" },
+    1: { title: "Flow Rate (L/s" }
+  },
+  series: {
+    0: { targetAxisIndex: 0 },
+    1: { targetAxisIndex: 1 },
+    2: { targetAxisIndex: 0 }
+  }
+}
```

Comment:
Added these examples, taken from issue #459. When I run `rackup`, they appear to be highlighted correctly. 

I'm not seeing the same error highlighting that appears in the JavaScript lexer.

<img width="643" alt="cfscript-lexer-highlight" src="https://cloud.githubusercontent.com/assets/5348419/16041767/4a313102-3205-11e6-93b8-227056dffc31.png">